crates/symbolicator-service/src/caching/config.rs

```diff
@@ -0,0 +1,35 @@
+use std::fmt;
+
+/// All known cache names.
+#[derive(Debug, Clone, Copy)]
+pub enum CacheName {
+    Objects,
+    ObjectMeta,
+    Auxdifs,
+    Il2cpp,
+    Symcaches,
+    Cficaches,
+    PpdbCaches,
+    Diagnostics,
+}
+
+impl AsRef<str> for CacheName {
+    fn as_ref(&self) -> &str {
+        match self {
+            Self::Objects => "objects",
+            Self::ObjectMeta => "object_meta",
+            Self::Auxdifs => "auxdifs",
+            Self::Il2cpp => "il2cpp",
+            Self::Symcaches => "symcaches",
+            Self::Cficaches => "cficaches",
+            Self::PpdbCaches => "ppdb_caches",
+            Self::Diagnostics => "diagnostics",
+        }
+    }
+}
+
+impl fmt::Display for CacheName {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.as_ref())
+    }
+}
```

Comment:
Do you expect this file/module to grow? Otherwise I would honestly put this in `mod.rs`.

Reply:
It might be possible to move more config related things here. It is also a bit of personal taste to keep files rather small and focused on a few things rather than have tons of stuff in a single huge file.